Doc/library/io.rst

```diff
@@ -1043,6 +1043,20 @@ Text I/O
       .. versionchanged:: 3.11
          The method supports ``encoding="locale"`` option.
 
+   .. method:: seek(cookie, whence, /)
+
+      Set the stream position and return the current position.
+
+      Four operations are supported,
+      given by the following argument combinations:
+
+      * ``seek(0, SEEK_SET)``: Rewind to the start of the stream.
+      * ``seek(n, SEEK_SET)``: Restore a previous position;
+        'n' is a number returned by :meth:`!tell`.
```

Comment:
Should it be `*x*` rather than `'x'`? (see the conventions used in other method docs above)

Reply:
> Should it be `*x*` rather than `'x'`? (see the conventions used in other method docs above)

\*x\* is the convention. I'll fix that.

> I think we should also be consistent in our use of the argument name -- either offset or cookie -- instead of using a new n; but I don't have a strong opinion here.

I thought about that earlier, but changed my mind. I'll reconsider it.

Reply:
Actually, the \*x\x convention is used for parameter names. Here, it was more of a code snippet (so, an *argument* to `seek()`). Perhaps double backticks are a better option.